File: OCH_Tao/tessoku-book/A07.py
from itertools import accumulate
D = int(input())
N = int(input())
X = [0]*(D+2)
for i in range(N):
  L, R = map(int, input().split())
  X[L] += 1
  X[R+1] -= 1
[print(i) for i in list(accumulate(X))[1:-1]]

File: OCH_Tao/tessoku-book/A07.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const D = parseInt(input.shift(), 10);
input.shift();
let X = Array(D + 1).fill(0);
for (const i of input) {
  const [L, R] = i.split(" ").map(x => parseInt(x, 10));
  X[L - 1]++;
  X[R]--;
}
let ans = [0];
for (let i = 0; i < D; i++) {
  ans.push(ans[i] + X[i]);
}
for (const i of ans.slice(1)) {
  console.log(i);
}